build/apis/contract-chain-c-chain-api.md

```diff
@@ -41,7 +41,459 @@ where `blockchainID` is the ID of the blockchain running the EVM.
+
+Get the private key that controls a given address. The returned private key can be added to a user with `avax.importKey`.
+
+#### Signature
+
+```go
+avax.exportKey({
+    username: string,
+    password:string,
+    address:string
+}) -> {privateKey: string}
+```
+
+##### Request
+
+* `username` must control `address`.
+* `address` is the address for which you want to export the corresponding private key. It should be in hex format.
+
+##### Response
+
+* `privateKey` is the CB58 endcoded string representation of the private key that controls `address`. It has a `PrivateKey-` prefix and can be used to import a key via `avax.importKey`.
+* `privateKeyHex` is the hex string representation of the private key that controls `address`. It can be used to import an account in to Metamask.
+
+#### Example Call
+
+```json
+curl -X POST --data '{
+    "jsonrpc":"2.0",
+    "id"     :1,
+    "method" :"avax.exportKey",
+    "params" :{
+        "username" :"myUsername",
+        "password":"myPassword",
+        "address": "0xc876DF0F099b3eb32cBB78820d39F5813f73E18C"
+    }
+}' -H 'content-type:application/json;' 127.0.0.1:9650/ext/bc/C/avax
+```
+
+#### Example Response
+
+```json
+{
+    "jsonrpc": "2.0",
+    "result": {
+        "privateKey": "PrivateKey-2o2uPgTSf3aR5nW6yLHjBEAiatAFKEhApvYzsjvAJKRXVWCYkE",
+        "privateKeyHex": "0xec381fb8d32168be4cf7f8d4ce9d8ca892d77ba574264f3665ad5edb89710157"
+    },
+    "id": 1
+}}
+```
+
+### avax.getUTXOs
+
+Gets the UTXOs that reference a given address.
+
+#### **Signature**
+
+```text
+avax.getUTXOs(
+    {
+        addresses: string,
+        limit: int, (optional)
+        startIndex: { (optional)
+            address: string,
+            utxo: string
+        },
+        sourceChain: string,
+        encoding: string, (optional)
+    },
+) -> 
+{
+    numFetched: int
+    utxos: []string,
+    endIndex: {
+        address: string,
+        utxo: string
+    }
+}
+```
+
+* `utxos` is a list of UTXOs such that each UTXO references at least one address in `addresses`.
+* At most `limit` UTXOs are returned. If `limit` is omitted or greater than 1024, it is set to 1024.
+* This method supports pagination. `endIndex` denotes the last UTXO returned. To get the next set of UTXOs, use the value of `endIndex` as `startIndex` in the next call.
+* If `startIndex` is omitted, will fetch all UTXOs up to `limit`.
+* When using pagination \(ie when `startIndex` is provided\), UTXOs are not guaranteed to be unique across multiple calls. That is, a UTXO may appear in the result of the first call, and then again in the second call.
+* When using pagination, consistency is not guaranteed across multiple calls. That is, the UTXO set of the addresses may have changed between calls.
+* `encoding` sets the format for the returned UTXOs. Can be either “cb58” or “hex”. Defaults to “cb58”.
+
+#### **Example**
+
+Suppose we want all UTXOs that reference at least one of `C-avax1yzt57wd8me6xmy3t42lz8m5lg6yruy79m6whsf`.
```

Comment:
This says `Suppose we want all UTXOs that reference` but actually it will only fetch at most 3 UTXOs since we set `limit` to 3, so let's either remove the limit or this line